src/modules/Dashboard/features/Participants/Participants.styles.ts

```diff
@@ -0,0 +1,84 @@
+import { Button, styled } from '@mui/material';
+
+import { DashboardTable } from 'modules/Dashboard/components';
+import { Search } from 'shared/components';
+import { StyledIcon } from 'shared/components/Search/Search.styles';
+import { StyledFlexSpaceBetween, StyledFlexTopCenter } from 'shared/styles/styledComponents';
+import theme from 'shared/styles/theme';
+import { variables } from 'shared/styles/variables';
+import { shouldForwardProp } from 'shared/utils/shouldForwardProp';
+
+import { ParticipantsColumnsWidth } from './Participants.const';
+
+export const ParticipantsHeader = styled(StyledFlexSpaceBetween, shouldForwardProp)`
+  margin-bottom: ${theme.spacing(2.4)};
+  min-width: 805px;
+  font-size: ${variables.font.size.lg};
+`;
+
+export const HeaderSectionLeft = styled(StyledFlexTopCenter, shouldForwardProp)`
+  gap: ${theme.spacing(1.2)};
+  width: 50%;
+`;
+
+export const HeaderSectionRight = styled(StyledFlexTopCenter, shouldForwardProp)`
+  gap: ${theme.spacing(1.2)};
+  width: 50%;
+  justify-content: flex-end;
+`;
+
+export const ParticipantSearchButton = styled(Search)`
+  flex: 0 0 50%;
+  min-width: 320px;
+`;
+
+export const StyledButton = styled(Button)`
+  && {
+    svg {
+      fill: ${variables.palette.on_surface_variant};
+    }
+  }
+`;
+
+export const FiltersButton = styled(StyledButton)`
+  color: ${variables.palette.on_surface_variant};
+`;
+
+export const SortByButton = styled(StyledButton)`
+  color: ${variables.palette.on_surface_variant};
+`;
+
+export const AddParticipantButton = styled(StyledButton)`
+  min-width: 137px;
+`;
+
+export const StyledCheckBox = styled(StyledIcon)`
+  svg {
+    fill: ${variables.palette.outline_variant};
+  }
+`;
+
+export const ParticipantsTable = styled(DashboardTable)`
+  th:nth-child(1) {
+    padding-left: 24px;
+  }
+
+  td:nth-child(1) {
+    padding-left: 14px;
+  }
+
+  td,
+  th {
+    min-width: 130px;
+  }
```

Comment:
Just noting that many lengths in this file are specified in `px`, but if you search the code, `rem` is primarily used throughout the app. Whether or not it will currently impact anything is uncertain, but the beauty of `rem` is that you can use it to scale the whole site via a root-level `font-size` change (if that's ever desired for a given breakpoint, perhaps in the future – handy for widescreen breakpoints, for instance). The conversion to `rem` is easy since `10px == 1rem`.

Reply:
Good call. I think that's a worthwhile change. Thanks for the pointer